manifests/t/th-ch/YouTubeMusic/1.6.2/th-ch.YouTubeMusic.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: th-ch.YouTubeMusic
+PackageVersion: 1.6.2
+InstallerType: nullsoft
+Commands:
+- youtube-music
+ReleaseDate: "2020-11-22"
+Installers:
+- Architecture: x86
```

Comment:
```suggestion
- Architecture: x64
```

Reply:
Thanks for the context! Will make the updates to the relevant PRs.

Reply:
Thanks! I've pushed the fix now.